### DISCUSSION TITLE
In cluster mode, Raft log size grows rapidly

### DISCUSSION THREAD

**Discussion by coolbeevip** [apache/iotdb#4925] [Q&A] [answered]
What is the compaction and cleanup strategy for Raft Log? 

If the Raft Log size can be controlled within a small size and use SSD disk storage Raft Log, can the cluster throughput be improved?

**Comment by jixuan1989** [comment]
@LebronAl

**Comment by OneSizeFitsQuorum** [comment, marked as answer]
You can use these two parameters `max_number_of_persist_raft_log_files` and `max_raft_log_persist_data_size_per_file` in [iotdb-cluster.properities](https://github.com/apache/iotdb/blob/master/cluster/src/assembly/resources/conf/iotdb-cluster.properties#L165) to control the size of the raftlog, as described in the comments in the configuration file.
